src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -32,6 +32,7 @@ const BlogPost = ({ data }) => {
 
     return () => {
       window.removeEventListener('scroll', handleScroll);
+      setState(BannerType.home);
```

Comment:
Why do you not set this in an useEffect in the home?

Reply:
Because the home doesn't have a useEffect, but, I can create it

Reply:
I tried to create it, but, it didn't work.